home/models.py

```diff
@@ -102,6 +109,10 @@ class Answer(models.Model):
     def __str__(self):
         return self.content
 
+    @classmethod
+    def get_Answers_by_date(cls):
```

Comment:
please stick to the usual Python coding conventions of having method names be fully lower case